Automattic/wp-calypso
# Set default A records on user request

#### Changes proposed in this Pull Request

* Add the capability to set the default A records and remove an existing root A/AAAA records.

#### Testing instructions

Set non-default A and AAAA records on a domain.
You can also set the pointsToWpcom value for the test domain to `false` to make it easier to test.

In the DNS menu, use the action button at the top of page and select the option to reset the default A records.
Make sure that you get the success message and that the page is re-rendered with the new records.
Also use the patch instructions to fake an error and make sure that the error message is shown and the rendered records are unchanged.

Also make sure that the deleted records and the added records are in the same changeset.
